.bazelrc

```diff
@@ -11,23 +11,23 @@ build --experimental_ui_deduplicate
 # This workaround is needed to prevent Bazel from compiling the same file twice (once PIC and once not).
 build:linux --force_pic
 build:macos --force_pic
+build:clang-cl --compiler=clang-cl
+build:msvc --compiler=msvc-cl
 # TODO(mehrdadn): Revert the "-\\.(asm|S)$" exclusion when this Bazel bug
 #                 for compiling assembly files is fixed on Windows:
 #                 https://github.com/bazelbuild/bazel/issues/8924
 # Warnings should be errors
-build --per_file_copt="-\\.(asm|S)$,-.*/arrow/util/logging\\.cc@-Werror"
+build:linux    --per_file_copt="-\\.(asm|S)$,-.*/arrow/util/logging\\.cc@-Werror"
+build:macos    --per_file_copt="-\\.(asm|S)$,-.*/arrow/util/logging\\.cc@-Werror"
+build:clang-cl --per_file_copt="-\\.(asm|S)$,-.*/arrow/util/logging\\.cc@-Werror"
+#build:msvc     --per_file_copt="-\\.(asm|S)$,-.*/arrow/util/logging\\.cc@-WX"
```

Comment:
why is this line comment out?

Reply:
MSVC produces warnings right now that aren't trivial to address, and we don't want to fail the build just because of that.